static/docs/commands-reference/move.md

```diff
@@ -41,10 +41,10 @@ The `dvc add` command would create a `data.csv.dvc` DVC-file with the following
 content:
 
 ```yaml
-md5: 3d1a3e5a5b662490e198d6a6ae84984b
+md5: df33f383592e7a399e106ba1cb487a8c
+wdir: .
 outs:
-  - cache: true
```

Comment:
let's keep it as it was before. Where did you take data.csv to play with?

Reply:
created a csv file with few records.